src/App.tsx

```diff
@@ -44,37 +45,46 @@ const App = () => {
   }
 
   if (!innholdsinformasjon) {
-    return !URL.includes('oria') ? <Header /> : null;
+    return !URL.includes(oriaKeyword) ? <Header /> : null;
   }
 
   return (
     <>
-      {!URL.includes('oria') && <Header />}
+      {!URL.includes(oriaKeyword) && <Header />}
       {isLoading ? (
         <progress />
       ) : (
         <>
-          <TitleLabel>{innholdsinformasjon.title}</TitleLabel>
+          <TitleLabel className={URL.includes(oriaKeyword) ? oriaKeyword : ''}>{innholdsinformasjon.title}</TitleLabel>
           {imageUrl && innholdsinformasjon.image_small && (
-            <ImageContainer src={imageUrl + innholdsinformasjon.image_small} alt="Bilde av boken" />
+            <ImageContainer
+              className={URL.includes(oriaKeyword) ? oriaKeyword : ''}
+              src={imageUrl + innholdsinformasjon.image_small}
+              alt="Bilde av boken"
+            />
           )}
-          <ISBNLabel>ISBN: {innholdsinformasjon.isbn}</ISBNLabel>
+          <ISBNLabel className={URL.includes(oriaKeyword) ? oriaKeyword : ''}>
+            ISBN: {innholdsinformasjon.isbn}
+          </ISBNLabel>
           {!isEmpty(innholdsinformasjon.description_short) && (
             <CollapsedBox
+              className={URL.includes(oriaKeyword) ? oriaKeyword : ''}
               name="Beskrivelse fra forlaget (kort)"
               contents={innholdsinformasjon.description_short}
               open={true}
             />
           )}
           {!isEmpty(innholdsinformasjon.description_long) && (
             <CollapsedBox
+              className={URL.includes(oriaKeyword) ? oriaKeyword : ''}
               name="Beskrivelse fra forlaget (lang)"
               contents={innholdsinformasjon.description_long}
               open={!isEmpty(innholdsinformasjon.description_long)}
```

Comment:
Litt pirkete og en smakssak, men jeg mener siden ser ryddigere ut om bare en av kort/lang description er åpen når man ankommer siden, hvis begge er åpen så strekker de seg ofte ut slik at man må skrolle, pluss at man blir presenter med veldig mye tekst.

Reply:
Jeg har så vidt jeg vet bare fulgt den logikken som var der og ikke tenkt så mye over hva som er best her. Men er veldig åpen for at jeg kan ha endret logikken uforvarende og/eller at det burde være som du sier :D

Reply:
Tror det var slik før (usikker, lenge siden). Men du kan teste det ut å se hva du liker, burde bare være å bytte ut open={!isEmpty(innholdsinformasjon.description_long)} med open={isEmpty(innholdsinformasjon.description_short)}

Reply:
Du hadde rett, jeg hadde "oversatt" feil og logikken ble ikke riktig. Har fikset det :)